docs/_static/iframe.js

```diff
@@ -0,0 +1,11 @@
+function inIframe() {
+    try {
+      return window.self !== window.top;
+    } catch (Exception) {
```

Comment:
I think in Javascript we `catch (err)`.
It can't match by class.

Reply:
updated